train_model.py

```diff
@@ -1,27 +1,91 @@
-'''This file holds the training loops for the KiU-Net model found in model.py
+'''This file holds the training loops for KiU-Net and
+U-Net found in model.py
 
-Class: CISC867-Deep Learning
-Author: Henry Lee
+Author: Henry Lee. Nicole Kitner
 Date: 2021-10-07
 '''
 import tensorflow as tf
 from tensorflow import keras
-from model.py import model as kiunet
-# insert import from data_processing #
+from tensorflow.keras.callbacks import TensorBoard
+from model import kiunet_model, unet_model
```

Comment:
from model import kiunet, unet_from_code

(but assuming we change unet_from_code to just unet it would be unet instead)